static/app/components/stream/group.spec.tsx

```diff
@@ -14,10 +14,10 @@ import {trackAnalytics} from 'sentry/utils/analytics';
 
 jest.mock('sentry/utils/analytics');
 
-describe('StreamGroup', function () {
+describe('StreamGroup', () => {
```

Comment:
Something was wrong with my formatter, it converted all these to arrow functions. will try to revert those changes